### ISSUE TITLE
Ideas to make things more realistic 

### ISSUE THREAD

**Issue by targerian** [finndev/PokeBuddy#1069]
Hello   o/ 
I've been thinking about some tricks that would might make pokebuddy more realistic, things like.
 A break handler.. with random amount of breaks time between a 
random times
for example " take a break between "3 to 5" minutes every "60 to 180 minute.
Something to make bot switch between "normal. good, great and excellent throws" 
Something that doesn't let the bot transfer bad pokemons as soon as it catches them.. more like waiting for a random amount of them then transfer them all at once..
regards.

**Comment by Demolition1986** [comment]
Yea wanna make it, so simple. Install some fake gps on ur mobile and do a manually farm like that, it's called manual touch.. Btw be awared that i think Inantic installed some type of program in server that detects some dll of bots, cause that ppl are getting banned.

**Comment by targerian** [comment]
IF that realy happenes you would've seen all the botters banned. I don't see so.. Im seeing people who survived that ban wave. it also be might one ot the best awesome tricky moves if they really know who's really botting, but they didn't ban all we you don't know the main criteria and conditions about how they ban!

**Comment by Synrise** [comment]
My main, played 3 weeks without bot, 3 days with bot, didn't get banned. My alt, 1 day of botting, from lvl 1 to 16, got banned. I think they look at how long you had the account and how fast you leveled up on it.

**Comment by targerian** [comment]
idk, it might be one of the criteria. but my friend "lv 25" got banned while i'm higher than him. and ,my account is newer than his! and he's banned

**Comment by Demolition1986** [comment]
Can I tell you something?  

Ill give u an exemple, if your friend that was bannned, he was lv high and with good pokemon +cp and all the time he took gyms and let his pokemon as master of gym, there normal players or maybe hack players who cares, can report him and then with a lot reports he will be investigate by Inantic and maybe he was banned cause that.. Old players +lv that was banned with high lv and maybe not boting,  i sure 100% that maybe was reported and got investigation.

Now if you try to bot in these days, I supose is hard and lucky to you dont get a ban or hardban using bots. Is my opinion.

**Comment by targerian** [comment]
Sounds reasonable. I shall take this in my considerations. specially.. I haven't put any of my pokemons inside a gym yet.

**Comment by Demolition1986** [comment]
@targerian Yeah dude, u know a lot of people just bot to improve a 3k cp pokemon and then go to fight at gym.. But dont you forget that razonable people will know when you are cheater or not.. :P In my country Argentina for exemple, pokemon was launched 04/08 today is 13/08 and how is possible to see people around map lv 36 with pokemon +3000cp as master of gym? 

Come on really bro? U wanna get reported? xD

**Comment by Demolition1986** [comment]
@targerian Btwi tell u that, but one day I took 7 differents gym in my zone, of course with no pokemon +3000cp, was pokemon 1,2 1.8, that seems more realistic. xD than took 7 gyms with 7 pokemons as master cp 2.9 to 3.k

**Comment by targerian** [comment]
you're people seems more kind than mine xP

**Comment by Demolition1986** [comment]
@targerian I tell more, to get no ban u need a good but really good setting to be like human move, or better go for a fake gps and farm manually.. No way to commit a mistakes.
